packages/themes/src/default.ts

```diff
@@ -168,7 +168,15 @@ export const defaultTheme = {
         '900': '#90722f',
       },
     },
-    gradient: 'linear-gradient(90deg, #0072EC 0%, #4e00c6 163.65%, #8c18e2 191.59%)',
```

Comment:
Glad we have these gradients defined now. Is it a smell that we have to refer to the actual hex value, #0072EC, rather than using `neutral.500`?

Reply:
I can update it to reference the neutral color, it would require some configuring to make neutral referenceable as we cant reference the theme object itself here.